library/pluginsystem/PluginSystem.php

```diff
@@ -76,7 +76,7 @@ public function init()
         $composerJsonPath = realpath( __DIR__."/../../composer.json" );
         $composerJson = file_get_contents( $composerJsonPath );
         if ( $composerJson === false ) {
-            throw new Exception( "$composerJsonPath/composer.json is not readable" );
+            throw new \Exception( "$composerJsonPath/composer.json is not readable" );
```

Comment:
Yes. The plugin that adds the video conferencing functionality is on my github right now. https://github.com/kchapple/vidyo